api-schema/src/main/java/org/stellar/anchor/api/sep/operation/Field.java

```diff
@@ -0,0 +1,13 @@
+package org.stellar.anchor.api.sep.operation;
+
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class Field {
```

Comment:
💡 I am interested in the reason to extract the class to separate files. Can you elaborate?

Reply:
This is mostly preference, but I found that prefixing the types with `Sep*` difficult to read when they can be grouped together into separate files instead.

Reply:
> Can you help to point out which classes have the type with Sep* prefix?

For example, Sep31TxnFieldSpecs can be moved into a class called Sep31Operation.

Reply:
> These classes are specific to the AssetInfo file format. IMO, it would keep the class namespace cleaner.

Will move this into AssetInfo inner class.